feature/platform/transceiver/tests/zr_logical_channels_test/zr_logical_channels_test.go

```diff
@@ -146,6 +148,25 @@ func validateEthernetChannelTelemetry(t *testing.T, otnChIdx, ethernetChIdx uint
 			want: oc.Assignment_AssignmentType_LOGICAL_CHANNEL.String(),
 		},
 	}
+	var assignmentIndexTestcase struct {
```

Comment:
can you define a struct type near the top:

type testcase struct {
	desc string
	got  any
	want any
}

Then around line 105, you can have:

tcs := []testcase {
	<testcases>
}

Then here, this just changes to:

assignmentIndexTestCase := testcase{
	desc: "Assignment: Index",
	got: ec.GetAssignment(0).GetIndex(),
	want: uint32(0),
})
if deviations.EthChannelAssignmentCiscoNumbering(dut) {
	assignmentIndexTestCase.want = uint32(1)
}
tcs = append(tcs, assignmentIndexTestCase)

Reply:
Sure, will do!